docs/semgrep-sc/sc-getting-started.md

```diff
@@ -21,135 +28,70 @@ Object.entries(frontMatter).filter(
-  </tr>
-  <tr>
-   <td>Scan target
-   </td>
-   <td>First-party code (your codebase or repository)
-   </td>
-   <td>Open source dependencies 
-   </td>
-  </tr>
-  <tr>
-   <td>Triage workflow
-   </td>
-   <td>Findings can be categorized as:
-<ul>
-<li>Ignored (to triage false positives)</li>
-<li>Closed (resolved) by refactoring code</li>
-<li>Removed</li>
-</ul>
-   </td>
-   <td>Findings can be categorized as:
-<ul>
-<li>New</li>
-<li>In progress</li>
-<li>Fixed</li>
-<li>Ignored</li>
-</ul>
-   </td>
-  </tr>
-  <tr>
-   <td>Remediation workflow
-   </td>
-   <td>Code refactoring
-   </td>
-   <td>Upgrading or removing the dependency, code refactoring
-   </td>
-  </tr>
-  <tr>
-   <td>Notification channels
-   </td>
-   <td>Jira, Slack, Email, Webhooks
-   </td>
-   <td>Slack and email
-   </td>
-  </tr></tbody>
-</table>
+See [Permissions in GitHub](/docs/semgrep-app/getting-started-with-semgrep-app/#permissions-for-github) to learn more about how Semgrep features use requested permissions in GitHub.
 
-## Language support
+</TabItem>
 
-Refer to [Supported languages](/docs/supported-languages#semgrep-supply-chain) to see all languages supported by Semgrep Supply Chain.
+<TabItem value='signin-gitlab'>
 
-## Limitations of Semgrep Supply Chain scans
+<PlatformSigninGitlab />
 
-* Semgrep Supply Chain supports scanning for transitive or indirect dependencies for all of its supported languages **except Java**. Findings are collected and displayed in **Semgrep App > Supply Chain**.
-* Semgrep Supply Chain detects transitivity only for **Python and JavaScript **languages.
-* Semgrep Supply Chain generates reachable findings only for direct dependencies.
+See [Permissions in GitLab](docs/semgrep-app/getting-started-with-semgrep-app/#permissions-for-gitlab) to learn more about how Semgrep features use requested permissions in GitLab.
 
-## Scanning a repository with Semgrep Supply Chain
+</TabItem>
 
-:::note
-Semgrep Supply Chain **supports monorepositories (monorepos)** by treating each subdirectory as its own repository. Findings are grouped under these repositories based on the lockfile or manifest file present in the subdirectory.
-:::
+</Tabs>
 
-You can run a dependency scan within Semgrep App.
+### Step 2: Adding or onboarding a new repository
 
-### Scanning with Supply Chain rules through Semgrep App
 
-:::tip
-This is the preferred method to enable and run Semgrep Supply Chain.
-:::
+<PlatformAddRepo />
 
-:::info Prerequisites
-To run Semgrep Supply Chain, you need the following:
-* A Semgrep App account. See [Signing in to Semgrep App](/docs/semgrep-app/getting-started-with-semgrep-app/#signing-in-to-semgrep-app).
-* A project (repository) with a lockfile to scan. The repository must have at least one **lockfile**. See [Scanning a new project](/docs/semgrep-app/getting-started-with-semgrep-app/#scanning-a-new-project).
-:::
+### Step 3: Enabling Semgrep Supply Chain scans from within Semgrep App
```

Comment:
Change `within` to `in` or do not use (as in this case it can be omitted). The `within` is more used with particular reference to the linearity of time or space etc. (Already researched this a couple of times in the past, see for example https://www.macmillandictionary.com/dictionary/british/within). Benefit is that it can be shorter without it.

```suggestion
### Step 3: Enabling Semgrep Supply Chain scans from Semgrep App
```